pkg/serverless/daemon/routes_test.go

```diff
@@ -95,7 +100,7 @@ func TestTraceContext(t *testing.T) {
 		DetectLambdaLibrary: func() bool { return false },
 	}
 	client := &http.Client{Timeout: 1 * time.Second}
-	body := bytes.NewBuffer([]byte(`{"start_time": "2019-10-12T07:20:50.52Z","headers": {"x-datadog-trace-id": ["1234"]},"payload": "{\"headers\":{\"x-datadog-parent-id\":\"1111\",\"x-datadog-trace-id\":\"2222\"}}"}`))
+	body := bytes.NewBuffer([]byte(`{"start_time": "2019-10-12T07:20:50.52Z","Headers": {"x-datadog-trace-id": "2222"}}`))
```

Comment:
Do we still want the start time in this payload? Should it just be a raw function invocation payload now, since we are no longer unmarshalling? (Same for the other tests in this file)

Reply:
good catch! ✅